src/main/java/employee/utility/EmployeeUtil.java

```diff
@@ -0,0 +1,49 @@
+package employee.utility;
+
+public class EmployeeUtil {
```

Comment:
This is not a utility for employee, it's a utility for processing the headers for Employees, the name is misleading.

Reply:
done